packages/hooks/src/__tests__/useStatesList.spec.tsx

```diff
@@ -1,15 +1,15 @@
-import * as React from 'react';
+import React from 'react';
 import { renderHook } from '@testing-library/react-hooks';
-import { APIProvider, useFetch } from '@deriv/api';
+import { APIProvider, useQuery } from '@deriv/api';
```

Comment:
why did we change `useFetch` to `useQuery` ?

Reply:
useFetch is deprecated. All those imports should be replaced with useQuery